pandas/formats/style.py

```diff
@@ -444,15 +444,17 @@ def apply(self, func, axis=0, subset=None, **kwargs):
 
         Parameters
         ----------
-        func: function
-        axis: int, str or None
+        func : function
+            ``func`` should take a Series or DataFrame (depending
+            on ``axis``), and return an object with the same shape
+        axis : int, str or None
             apply to each column (``axis=0`` or ``'index'``)
             or to each row (``axis=1`` or ``'columns'``) or
-            to the entire DataFrame at once with ``axis=None``.
-        subset: IndexSlice
+            to the entire DataFrame at once with ``axis=None``
+        subset : IndexSlice
```

Comment:
I am wrong - it does specify spaces on both sides
(contrast this to linting which doesn't have s space - but different issue)